src/handlers/mentions.rs

```diff
@@ -95,23 +109,30 @@ pub(super) async fn handle_input(
         IssueData::load(&mut client, &event.issue, MENTIONS_KEY).await?;
     // Build the message to post to the issue.
     let mut result = String::new();
-    for to_mention in &input.paths {
-        if state.data.paths.iter().any(|p| p == to_mention) {
-            // Avoid duplicate mentions.
-            continue;
-        }
-        let MentionsPathConfig { message, cc } = &config.paths[to_mention];
-        if !result.is_empty() {
-            result.push_str("\n\n");
-        }
-        match message {
-            Some(m) => result.push_str(m),
-            None => write!(result, "Some changes occurred in {to_mention}").unwrap(),
+    match input {
+        MentionsInput::HasBorsCommit => {
+            result = config.bors_commit_message.to_string();
```

Comment:
I think this doesn't provide a way to opt-out and get the real message (e.g., if you are pulling in miri or clippy, and making changes as part of the merge to other files).

I think one option is to escape all of the mentions and still post the message - that gives the author the ability to re-ping the right folks. Though only if they're a member of the org for teams.

I think a better option is to detect the bors merge commits and then set the PR as draft and bail out. (Probably leaving a comment is still reasonable). That means that the author can just close the PR if they don't want to ping folks or fix it and then mark as ready (which should already retrigger the logic if I'm remembering right).

Reply:
That seems quite annoying for the cases in which this is intentional - the current version will warn but at least it doesn't require the PR author to take any additional steps if that's what they intended to do

Reply:
The additional step is clicking in the UI on ready for review - that feels pretty lightweight to me? It's also not strictly required - e.g., bors will listen to you in draft state iirc.

Reply:
Hmm, looking at the github API, I don't think we can mark someone else's pull request as being a draft. Escaping the pings is possible though, I'll update

Reply:
It seems like the conversion is possible, just needs graphql rather than the rest API. 

https://docs.github.com/en/graphql/reference/mutations#convertpullrequesttodraft